index.ts

```diff
@@ -43,7 +53,7 @@ const reassignEventListeners = (ws, oldWs, listeners) => {
 const ReconnectingWebsocket = function(
     url: string,
     protocols?: string|string[],
-    options: Object = {}
+    options = <Options>{}
```

Comment:
Not really... The first one you first declare the expected type and then try to assign the value to it (with the possible typing errors) and the second one just cast the object to the <type> and the options type will be inferred.